app/v1/models/response/listPayments/Payment.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.models.response.listPayments
+
+import play.api.libs.json.{JsPath, Json, OWrites, Reads}
+
+case class Payment(id: String, amount: BigDecimal, method: String, transactionDate: String)
+
+object Payment {
+  implicit val reads: Reads[Payment] = for {
+    paymentLot <- (JsPath \ "paymentLot").read[String]
+    paymentLotItem <- (JsPath \ "paymentLotItem").read[String]
+    amount <- (JsPath \ "amount").read[BigDecimal]
+    method <- (JsPath \ "method").read[String]
```

Comment:
The DES fields for these should be paymentAmount and paymentMethod (as per the DES spec). The technical spec appears to be wrong in this case.